packages/core/src/CorePlugin.ts

```diff
@@ -1,18 +1,27 @@
 import { JWPlugin } from './JWPlugin';
 import JWEditor from './JWEditor';
-import { VRangeDescription, Direction, RelativePosition } from './VRange';
+import { VRangeDescription, Direction, RelativePosition, RangePosition } from './VRange';
 import { VNode } from './VNode';
 
-export interface InsertParams {
+export interface RangeFromTo {
+    start: VNode;
+    startPosition?: RangePosition;
+    end?: VNode;
+    endPosition?: RangePosition;
+}
```

Comment:
We already have an interface that is very similar to this one, except that it has `direction` as well. Couldn't we just put `direction` as optional ? Check if there are parts of the code where we make the assumption that `direction` is always there.